cmd/load-runner/main.go

```diff
@@ -0,0 +1,171 @@
+package main
+
+import (
+	"context"
+	"flag"
+	"fmt"
+	"os"
+	"os/exec"
+	"os/signal"
+	"regexp"
+	"time"
+)
+
+type config struct {
+	duration       string
+	parsedDuration time.Duration
+	namespace      string
+	loadName       string
+	chartFilesPath string
+}
+
+const (
+	defaultReleaseNamePrefix = "rudder-load"
+	defaultChartFilesPath    = "./artifacts/helm"
+)
+
+func main() {
+	cfg := parseFlags()
+	if err := validateInputs(cfg); err != nil {
+		fmt.Fprintf(os.Stderr, "Validation error: %v\n", err)
+		os.Exit(1)
+	}
+
+	if err := run(cfg); err != nil {
+		fmt.Fprintf(os.Stderr, "Error: %v\n", err)
+		os.Exit(1)
+	}
+}
+
+func parseFlags() *config {
+	var cfg config
+	flag.StringVar(&cfg.duration, "d", "", "Duration to run (e.g., 1h, 30m, 5s)")
```

Comment:
[optional] Also here I'd use the go-kit config. It's common enough in our codebase and it offers a few utilities. For example in this case you won't have to parse the duration, it's done by the library.

Reply:
go-kit config seems to working with env variables not sure how can I use it with command line args. Let me know if this can be tackled later.